clients/python/tensorzero/tensorzero.pyi

```diff
@@ -55,10 +56,13 @@ class StoredInference:
         variant_name: str,
         input: Any,
         output: Any,
-        episode_id: Any,
-        inference_id: Any,
+        episode_id: UUID,
+        inference_id: UUID,
         tool_params: Optional[Any] = None,
         output_schema: Optional[Any] = None,
+        dispreferred_outputs: Union[
+            List[List[ContentBlock]], List[JsonInferenceOutput]
```

Comment:
Can you make the `List[List[ContentBlock]]` a type alias, and explain the doubly-nested list?